tmt/utils.py

```diff
@@ -6120,30 +6122,40 @@ def regex_search(
     return environment
 
 
-def render_template_file(
-        template_filepath: Path,
+def render_template(
+        template_str: str,
         environment: Optional[jinja2.Environment] = None,
         **variables: Any
         ) -> str:
```

Comment:
Could this function accept an optional template file path? It would improve the exceptions: `f"Could not parse template '{template_filepath}' at line {exc.lineno}."` is more helpful then `f"Could not parse template at line {exc.lineno}."`, I would know what template was causing the trouble. It would be an optional input, if it's not set, the best we can do is the latter error message, but if it's available, it would be a nice hint.

Reply:
Added in 78d7217caecb8c3bd05d92445264de9de4bfaf02.